src/menus/PluginMenus.cpp

```diff
@@ -32,6 +32,383 @@
 // private helper classes and functions
 namespace {
 
+using MenuList = std::vector<std::pair<TranslatableString, std::vector<TranslatableString>>>;
+
+const MenuList& PreferredEffectsMenu()
+{
+   static MenuList items =
+   {
+      {
+         XO("Volume and compression"),
+         {
+            XO("Amplify"),
+            XO("Compressor"),
+            XO("Limiter"),
+            XO("Normalize"),
+            XO("Loudness Normalization"),
+            XO("Auto Duck")
+         }
+      },
+      {
+         XO("Fading"),
+         {
+            XO("Fade In"),
+            XO("Fade Out"),
+            XO("Studio Fade Out"),
+            XO("Adjustable Fade"),
+            XO("Crossfade Clips"),
+            XO("Crossfade Tracks")
+         }
+      },
+      {
+         XO("Pitch and Tempo"),
+         {
+            XO("Change Pitch"),
+            XO("Change Speed"),
+            XO("Change Tempo"),
+            XO("Paulstretch"),
+            XO("Sliding Stretch"),
+            XO("Reverse")
+         }
+      },
+      {
+         XO("EQ and Filters"),
+         {
+            XO("Bass and Treble"),
+            XO("Graphic EQ"),
+            XO("Filter Curve EQ"),
+            XO("High-Pass Filter"),
+            XO("Low-Pass Filter"),
+            XO("Notch Filter")
+         }
+      },
+      {
+         XO("Noise Removal and Repair"),
+         {
+            XO("Click Removal"),
+            XO("Noise Reduction"),
+            XO("Noise Gate"),
+            XO("Repair"),
+            XO("Clip Fix")
+         }
+      },
+      {
+         XO("Delay and Reverb"),
+         {
+            XO("Echo"),
+            XO("Reverb"),
+            XO("Delay")
+         }
+      },
+      {
+         XO("Distortion and Modulation"),
+         {
+            XO("Tremolo"),
+            XO("Distortion"),
+            XO("Wahwah"),
+            XO("Phaser"),
+            XO("Vocoder")
+         }
+      },
+      {
+         XO("Special"),
+         {
+            XO("Repeat"),
+            XO("Invert"),
+            XO("Truncate Silence"),
+            XO("Vocal Reduction and Isolation"),
+            XO("Vocal Remover")
+         }
+      },
+      {
+         XO("Spectral tools"),
+         {
+            XO("Spectral Delete"),
+            XO("Spectral edit multi tool"),
+            XO("Spectral edit parametric EQ"),
+            XO("Spectral edit shelves")
```

Comment:
Same with these guys: 
* Spectral Tools
* Spectral Delete
* Spectral Edit Multi Tool
* Spectral Edit Parametric EQ
* Spectral Edit Shelves

Reply:
@LWinterberg as Paul said in his comment above these are "identifiers", not names, but "Spectral Tools" is a group name and I'll fix that

Reply:
I VETO this change request as part of the same issue.

Make this change request a different, minor issue.

Changing the name while preserving macro compatibility turns out not to be as easy for Nyquist effects as it would be for Invert.  It needs more thought.